design/media_types.go

```diff
@@ -72,6 +72,12 @@ var meta = a.Type("workItemListResponseMeta", func() {
 	a.Required("totalCount")
 })
 
+var position = a.Type("workItemReorderPosition", func() {
+	a.Attribute("above", d.String)
```

Comment:
Please, add a description to this attribute and the type. What is the `above` string? Is it the work item ID of the work item above after or before re-ordering? Good documentation of the generated swagger interface helps everybody in the long run. Here's the swagger definition for your PR so far: http://swagger.goa.design/?url=github.com%2FDhritiShikhar%2Falmighty-core%2Fdesign%40orderofexecution1# .